### PR TITLE
Remove unuseful toString method

### PR DESCRIPTION
**Pull request by deadwind4**
### What changes were proposed in this pull request?

Remove some toString() method.

### Why are the changes needed?

Some toString() method is unuseful, so they should be deleted.

### How was this patch tested?

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Gentle ping, @deadwind4

File: java/tools/src/java/org/apache/orc/tools/FileDump.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ public static void main(Configuration conf, String[] args) throws Exception {
         rowIndexCols = null; // All the columns
       } else {
         String[] colStrs = cli.getOptionValue("r").split(",");
-        rowIndexCols = new ArrayList<Integer>(colStrs.length);
```
Comment (dongjoon-hyun) [inline java/tools/src/java/org/apache/orc/tools/FileDump.java:106]: The same mistake like https://github.com/apache/orc/pull/1181#discussion_r924633593

File: java/tools/src/java/org/apache/orc/tools/FileDump.java [hunk 2]
```diff
@@ -119,7 +119,7 @@ public static void main(Configuration conf, String[] args) throws Exception {
     }
 
     // if the specified path is directory, iterate through all files and print the file dump
-    List<String> filesInPath = new ArrayList<>();
```
Comment (dongjoon-hyun) [inline java/tools/src/java/org/apache/orc/tools/FileDump.java:122]: The same mistake like https://github.com/apache/orc/pull/1181#discussion_r924633593

**Comment by dongjoon-hyun** [comment]
Thank you, @deadwind4 . Mostly, the PR looks good.

cc @williamhyun , too

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
While reviewing this, I realized that this PR is incomplete.
Let me create a complete one with coauthorship with you and me.

This PR has 8 line changes which is less than 30% of all occurrence.
```
$ git diff --stat             
 java/core/src/java/org/apache/orc/Reader.java                        |  2 +-
 java/core/src/java/org/apache/orc/impl/OutStream.java                | 13 +++++--------
 java/core/src/java/org/apache/orc/impl/ParserUtils.java              |  4 ++--
 java/core/src/java/org/apache/orc/impl/RecordReaderUtils.java        |  2 +-
 java/core/src/java/org/apache/orc/impl/SchemaEvolution.java          |  4 ++--
 java/core/src/java/org/apache/orc/util/BloomFilter.java              |  2 +-
 java/tools/src/java/org/apache/orc/tools/FileDump.java               |  8 ++++----
 java/tools/src/java/org/apache/orc/tools/PrintData.java              |  3 +--
 java/tools/src/java/org/apache/orc/tools/json/HiveType.java          |  2 +-
 java/tools/src/java/org/apache/orc/tools/json/JsonSchemaFinder.java  |  2 +-
 java/tools/src/test/org/apache/orc/tools/convert/TestJsonReader.java | 12 ++++++------
 11 files changed, 25 insertions(+), 29 deletions(-)
```

**Comment by dongjoon-hyun** [comment]
Here is a superset of this PR, @deadwind4 .
ORC-1219 will be assigned to you because you are the main contributor.

**Comment by deadwind4** [comment]
> Here is a superset of this PR, @deadwind4 . [ORC-1219](https://issues.apache.org/jira/browse/ORC-1219) will be assigned to you because you are the main contributor. Please let me know your Apache JIRA ID.

I have commented on this Jira ticket so that you can assign me. Thanks a lot.

**Comment by dongjoon-hyun** [comment]
Thank you!

**Pull request closed without merging**